node/sc/bridgepool/sorted_map_list.go

```diff
@@ -96,15 +103,21 @@ func (m *ItemSortedMap) Exist(nonce uint64) bool {
 
 // Put inserts a new item into the map, also updating the map's nonce
 // index. If a item already exists with the same nonce, it's overwritten.
-func (m *ItemSortedMap) Put(event itemWithNonce) {
+func (m *ItemSortedMap) Put(event itemWithNonce) error {
```

Comment:
This `error` is not handled.

Reply:
Do you mean the caller doesn't handle the error?
I thought we don't need to handle it. :)

Reply:
Thanks for the explanation! I understood your intention.